api/app/serializers/spree/v2/storefront/wishlist_serializer.rb

```diff
@@ -0,0 +1,14 @@
+module Spree
+  module V2
+    module Storefront
+      class WishlistSerializer < BaseSerializer
+        set_type :wishlist
+
+        attributes :token, :name, :is_private, :is_default
+
+        belongs_to :user
```

Comment:
@damianlegawiec 

I'm thinking we should remove the `belongs_to :user` in the serializer here.

**Reason:**
As I write the Open API docs I'm getting the feeling that we should not really expose the user and their `shipping`/`billing address`, `order_count` and `store_ credit` all through the Wishlists, especially as a Wishlist can be accessed publicly via the show action if set to `is_private: false`.

Reply:
@MatthewKennedy good call